### DISCUSSION TITLE
light dom css not supported two level web comopnent

### DISCUSSION THREAD

**Discussion by jitendraP-ashutec** [lit/lit#3544] [Q&A]
I have 2 component wb-tabs and web-tab like follow

```
<wb-tabs> 
     <wb-tab name="tab1">
                <div class="container"> This is will tab 1 content </div>
      <wb-tab>
      <wb-tab name="tab2">
                <div class="container"> This is will tab 2 content </div>
       <wb-tab>
</wb-tabs>
```

Issue I am facing is that what css style I have applied in wb-tab's  content is not applying.

But If I am writing  only wb-tab component like:

```
<wb-tab name="tab1">
                <div class="container"> This is will tab 1 content </div>
<web-tab>
```

then css/styles is applying for class 'container'. 

Does any one has idea about it why is not apply styles?

Thanks

**Comment by e111077** [comment]
If the styles are inside of wb-tab and not wb-tabs this should work. Slotted styles can only go one level down. Can you spin up an example in the [lit playground](https://lit.dev/playground) please?
